src/features/post/components/Post.tsx

```diff
@@ -162,9 +162,11 @@ export const Post: FC<PostProps> = ({post}: PostProps) => {
             <div data-testid="post-tags" className="mx-10 flex w-full py-10">
               <span className="mr-2">Tags : </span>
               <div className="flex justify-evenly">
-                <Badge className="mx-1">Lorem</Badge>
-                <Badge className="mx-1">Ipsum</Badge>
-                <Badge className="mx-1">Hello</Badge>
+                {post.categories?.map((category, index) => (
+                  <Badge data-testid={`category-${index + 1}`} className="mx-1">
```

Comment:
create custom Badge component for Category
to color the Badge based on the label text